docs/postgres_install.rst

```diff
@@ -0,0 +1,86 @@
+===============================
+RDataRetriever with PostgreSQL
```

Comment:
Since this is going in the main retriever documentation I think this header should probably be broadened to something like "Working with PostgreSQL".